src/renderer/Theming.js

```diff
@@ -0,0 +1,77 @@
+const React = require('react')
+const StyledThemeProvider = require('styled-components').ThemeProvider
+const { EventEmitter } = require('events')
+
+const defaultTheme = Object.freeze({
+  deltaPrimaryBg: '#415e6b',
+  deltaPrimaryBgLight: '#4c6e7d',
+  deltaPrimaryFg: '#ffffff',
+  deltaPrimaryFgLight: '#d0d0d0',
+  deltaHover: '#ececec',
+  deltaChatPrimaryFg: '#070c14',
+  deltaChatPrimaryFgLight: '#62656a',
+  deltaChatMessageBubbleSelf: '#efffde',
+  deltaChatMessageBubbleOther: '#ffffff',
+  deltaInfoMessageBubbleBg: '#000000',
+  deltaInfoMessageBubbleColor: 'white',
+  deltaChatMessageBubbleSelfStatusColor: '#4caf50',
+  deltaFocusBlue: '#42A5F5',
+  deltaSelected: '#4c6e7d' // same as deltaPrimaryBgLight
+})
+
+class ThemeManager extends EventEmitter {
```

Comment:
At least I don't really understand why we need a ThemeManager/EventEmitter

Reply:
Thats my quick 'n (maybe) dirty approach to make it changeable at runtime